packages/nextjs-demo/src/app/agent/page.tsx

```diff
@@ -0,0 +1,242 @@
+    // can't update the tts value once it's captured in closures.
+    const _tts = new AzureTextToSpeech(BuildUrl, AzureTextToSpeech.DEFAULT_VOICE, 1.2);
+    setManager(_manager);
+    setAsr(_asr);
+    setTts(_tts);
+    console.log('starting');
+    await _manager.startMic(100, () => handleStop());
+    _asr.addEventListener('transcript', (event: CustomEventInit<Transcript>) => {
+      const obj = event.detail!;
+      handleInputUpdate(obj.text, obj.final, _tts);
+      setInput(obj.text);
+    });
+    console.log('starting ASR');
+    _asr.start();
+    handleInputUpdate('Hi!', true, _tts);
+  };
+
+  return (
+    <>
+      <div className="w-full">
+        <p className="font-sm ml-2 mb-2">
+          This demo allows you to chat (via voice) with a drive-thru agent at a Krispy Kreme. Click Start to begin.
+        </p>
+        <div className="grid grid-cols-2">
+          <div className="p-4">
+            <p className="text-lg font-bold">🍩 DONUTS</p>
+            <ul className="text-sm">
+              <li>PUMPKIN SPICE ORIGINAL GLAZED® DOUGHNUT $1.29</li>
+              <ul>
+                <li>PUMPKIN SPICE CAKE DOUGHNUT $1.29</li>
+                <li>PUMPKIN SPICE CHEESECAKE SWIRL DOUGHNUT $1.29</li>
+                <li>PUMPKIN SPICE MAPLE PECAN DOUGHNUT $1.29</li>
+                <li>ORIGINAL GLAZED® DOUGHNUT $0.99</li>
+                <li>CHOCOLATE ICED GLAZED DOUGHNUT $1.09</li>
+                <li>CHOCOLATE ICED GLAZED DOUGHNUT WITH SPRINKLES $1.09</li>
+                <li>GLAZED RASPBERRY FILLED DOUGHNUT $1.09</li>
+                <li>GLAZED BLUEBERRY CAKE DOUGHNUT $1.09</li>
+                <li>STRAWBERRY ICED DOUGHNUT WITH SPRINKLES $1.09</li>
+                <li>GLAZED LEMON FILLED DOUGHNUT $1.09</li>
+                <li>CHOCOLATE ICED CUSTARD FILLED DOUGHNUT $1.09</li>
+                <li>CHOCOLATE ICED DOUGHNUT WITH KREME™ FILLING $1.09</li>
+                <li>CAKE BATTER DOUGHNUT $1.09</li>
+                <li>ORIGINAL GLAZED® DOUGHNUT HOLES $3.99</li>
+              </ul>
+            </ul>
+          </div>
+          <div className="p-4">
+            <p className="text-lg font-bold">☕️ COFFEE & DRINKS</p>
+            <ul className="text-sm">
+              <li>PUMPKIN SPICE COFFEE $2.59</li>
+              <li>PUMPKIN SPICE LATTE $4.59</li>
+              <li>CLASSIC BREWED COFFEE $1.79</li>
+              <li>CLASSIC DECAF BREWED COFFEE $1.79</li>
+              <li>LATTE $3.49</li>
+              <li>VANILLA SPECIALTY LATTE $3.49</li>
+              <li>ORIGINAL GLAZED® LATTE $3.49</li>
+              <li>CARAMEL SPECIALTY LATTE $3.49</li>
+              <li>CARAMEL MOCHA SPECIALTY LATTE $3.49</li>
+              <li>MOCHA SPECIALTY LATTE $3.49</li>
+            </ul>
+          </div>
+        </div>
+        <div>
+          <textarea className="m-2 w-full" readOnly id="output" rows={10} cols={80} value={output}></textarea>
```

Comment:
Minor: this is what the agent says, and not actual input, right? In that case, I'd use a `div` instead of `textarea`. https://tailwindui.com/components has examples you can copy/paste from. (Ask Zach for access if you don't already have it.)

This is only worth changing if it takes <5 minutes.